Format.pm

```diff
@@ -482,18 +506,25 @@ sub new
 
 ##----------------------------------------------------------------------
 
-=item round($number, $precision)
+=item round($number, $precision, $roundoption)
 
 Rounds the number to the specified precision.  If C<$precision> is
 omitted, the value of the C<DECIMAL_DIGITS> parameter is used (default
 value 2).  Both input and output are numeric (the function uses math
 operators rather than string manipulation to do its job), The value of
-C<$precision> may be any integer, positive or negative. Examples:
-
-  round(3.14159)       yields    3.14
-  round(3.14159, 4)    yields    3.1416
-  round(42.00, 4)      yields    42
-  round(1234, -2)      yields    1200
+C<$precision> may be any integer, positive or negative. If C<$roundoption>
+is omitted, the value of the C<ROUND_OPTION> paramter is used (default
+value 0). Examples:
+
+  round(3.14159)                  yields    3.14
+  round(3.14159, undef, 1)        yields    3.15
```

Comment:
change 1 & -1 to ROUND_xx constants
Document that undef means to use the default precision (DECIMAL_DIGITS setting).